element_interface/intan_loader.py

```diff
@@ -0,0 +1,271 @@
+                    "port_number": signal_group,
+                }
+                new_channel["native_channel_name"] = read_qstring(fid)
+                new_channel["custom_channel_name"] = read_qstring(fid)
+                (
+                    new_channel["native_order"],
+                    new_channel["custom_order"],
+                    signal_type,
+                    channel_enabled,
+                    new_channel["chip_channel"],
+                    command_stream,
+                    new_channel["board_stream"],
+                ) = struct.unpack(
+                    "<hhhhhhh", fid.read(14)
+                )  # ignore command_stream
+                new_trigger_channel = {}
+                (
+                    new_trigger_channel["voltage_trigger_mode"],
+                    new_trigger_channel["voltage_threshold"],
+                    new_trigger_channel["digital_trigger_channel"],
+                    new_trigger_channel["digital_edge_polarity"],
+                ) = struct.unpack("<hhhh", fid.read(8))
+                (
+                    new_channel["electrode_impedance_magnitude"],
+                    new_channel["electrode_impedance_phase"],
+                ) = struct.unpack("<ff", fid.read(8))
+
+                if channel_enabled:
+                    if signal_type == 0:
+                        header["amplifier_channels"].append(new_channel)
+                        header["spike_triggers"].append(new_trigger_channel)
+                    elif signal_type == 1:
+                        raise Exception("Wrong signal type for the rhs format")
+                        # header['aux_input_channels'].append(new_channel)
+                    elif signal_type == 2:
+                        raise Exception("Wrong signal type for the rhs format")
+                        # header['supply_voltage_channels'].append(new_channel)
+                    elif signal_type == 3:
+                        header["board_adc_channels"].append(new_channel)
+                    elif signal_type == 4:
+                        header["board_dac_channels"].append(new_channel)
+                    elif signal_type == 5:
+                        header["board_dig_in_channels"].append(new_channel)
+                    elif signal_type == 6:
+                        header["board_dig_out_channels"].append(new_channel)
+                    else:
+                        raise Exception("Unknown channel type.")
+
+    # Summarize contents of data file.
+    header["num_amplifier_channels"] = len(header["amplifier_channels"])
+    header["num_board_adc_channels"] = len(header["board_adc_channels"])
+    header["num_board_dac_channels"] = len(header["board_dac_channels"])
+    header["num_board_dig_in_channels"] = len(header["board_dig_in_channels"])
+    header["num_board_dig_out_channels"] = len(header["board_dig_out_channels"])
+
+    return header
+
+
+def load_rhs(folder: str, file_expr: str):
```

Comment:
```suggestion
def load_rhs(folder: str):
```

I don't think regex need to be specified here as an argument. This is supposed to fetch whatever that's inside the data folder

Reply:
It adds flexibility to not read everything. Say the files are `amp-B-000.dat ... to  amp-B-015.dat`, `amp-D-000.dat ... to  amp-D-015.dat` and `board-DIGITAL-IN-01.dat`. 

To read all channels (A and B) in the working directory:
load_rhs(".", "amp*")

To read only the `B` channels in the working directory:
load_rhs(".", "amp-B*")

Reply:
I see. Then we can keep that and set the default to e.g.,`*.dat` (read everything) and allow a user to specify if needed.